common/core/web/keyboard-processor/tests/cases/engine/context.js

```diff
@@ -736,16 +736,122 @@ var DEADKEY_STORE_TEST_4 = {
   }]
 };
 
+var NOTANY_NUL_TEST_1 = {
+  id: 4,
+  // Match condition for rule
+  rule: [{t: 'a', a:['a','b','c'], n: 1},{t:'a',a:['a', 'b', 'c']}],
```

Comment:
Can you add in a comment the equivalent kmn for the rule conditions, e.g. 

```
// store(abc) 'abc'
// notany(abc) any(abc)
```

Reply:
I've gone ahead and done this for every rule in the suite, even the older ones.